src/lib/stellar/transactions.js

```diff
@@ -331,3 +333,52 @@ export async function createPathPaymentStrictReceiveTransaction({
         network_passphrase: networkPassphrase,
     }
 }
+
+/**
+ * Constructs and returns a Stellar transaction for transferring assets to a contract or account.
+ * @async
+ * @function createContractTransferTransaction
+ * @param {Object} opts Options object
+ * @param {string} opts.source Public Stellar address to use as the source account of the transaction
+ * @param {string} opts.destination Public Stellar address or contract ID to receive the transfer
+ * @param {string} opts.amount Amount of the asset to transfer
+ * @param {string} opts.asset Asset to be transferred (example: USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5)
+ * @returns {Promise<TransactionResponse>} Object containing the relevant network passphrase and the built transaction envelope in XDR base64 encoding, ready to be signed and submitted
+ */
+export async function createContractTransferTransaction({ source, destination, amount, asset }) {
+    const server = new Server(horizonUrl);
+    const sourceAccount = await server.loadAccount(source);
+
+    const transaction = new TransactionBuilder(sourceAccount, {
+        networkPassphrase: networkPassphrase,
+        fee: maxFeePerOperation,
+    });
+
+    const [assetCode, assetIssuer] = asset.split(':');
+    const contractId = new Asset(assetCode, assetIssuer).contractId(Networks.TESTNET);
```

Comment:
Let's use the already-declared variable from the top of this file
```suggestion
    const contractId = new Asset(assetCode, assetIssuer).contractId(networkPassphrase);
```